src/routes/diagnostics.js

```diff
@@ -0,0 +1,56 @@
+module.exports = [
+    {
+        method: 'post',
+        path: '/api/v3/iofog/microservices/:id/imageSnapshot',
```

Comment:
change it to `/api/v3/iofog/microservices/:id/image-snapshot` and update swagger